tests/plugins/test_redmine.py

```diff
@@ -1,12 +1,8 @@
 # coding: utf-8
 """ Tests for the Redmine plugin """
 
-import time
-
-import pytest
-
-import did.base
-import did.cli
+# import did.base
+# import did.cli
```

Comment:
Why leave the commented out imports in here?

Reply:
Because of the disabled test at the bottom of the file. When the time comes to resurrect it, everything's there.